engine/src/main/java/org/terasology/rendering/dag/nodes/InitialPostProcessingNode.java

```diff
@@ -53,13 +51,12 @@
  * Stores the result into the InitialPostProcessingNode.INITIAL_POST_FBO, to be used at a later stage.
  */
 public class InitialPostProcessingNode extends AbstractNode {
-    public static final ResourceUrn INITIAL_POST_FBO = new ResourceUrn("engine:fbo.initialPost");
-    public static final ResourceUrn INITIAL_POST_MATERIAL = new ResourceUrn("engine:prog.initialPost");
+    static final ResourceUrn INITIAL_POST_FBO = new ResourceUrn("engine:fbo.initialPost");
```

Comment:
It can be, I'll just have to add a new suppress warning to make IntelliJ happy.

That being said, I'm hoping to make this (and all other similar variables) private, since like all the accessors, the code will ideally never access this directly.